pkg/preparer/orchestrate_test.go

```diff
@@ -82,7 +84,9 @@ func TestPreparerLaunchesPodsThatHaveDifferentSHAs(t *testing.T) {
 		currentManifest: existing,
 	}
 	newManifest := testManifest(t)
-	success := installAndLaunchPod(newManifest, testPod, logging.DefaultLogger)
+
+	var p *Preparer
```

Comment:
Agreed, it is a time bomb. We should be able to fix it sometime tomorrow, though.

After the reality store update and service registration get merged, the tests will fail regardless of the null pointer deref (because the requests to Consul will fail in installAndLaunchPod). I can fix the tests then. I mainly want to get the refactor into the repo quicker so that we can rebase our code onto it.